package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "uapi-json",
-  "version": "0.8.6",
+  "version": "0.8.8",
```

Comment:
https://www.npmjs.com/package/uapi-json
We had 0.8.7 somehow.
Stable not yet merged, because we are making branches from stable if we fix something.
And only then stable will be merged to master